src/Orchard.Web/Modules/Orchard.MediaLibrary.WebSearch/Views/EditorTemplates/Parts/WebSearch.BingWebSearchSettings.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@model Orchard.MediaLibrary.WebSearch.Models.BingWebSearchSettingsPart
+
+<fieldset>
+    <legend>@T("Bing Web Search")</legend>
+    <div>
+        @Html.TextBoxFor(m => m.ApiKey, new { @class = "text medium"})
```

Comment:
This should be handled as a password, like the SMTP-settings (same for Pixabay).